test/getUsersFromDatabase.js

```diff
@@ -0,0 +1,19 @@
+var request = require("request");
+var assert = require('assert');
+const getUserFromDatabase = require('../src/getUsersFromDatabase.js');
+
+describe('Get Users From Database', async ()=> {
+    it('get user brian', function () {
+        const user = getUserFromDatabase.getUserFromDatabase('brian')
+        .then(function(data){
+           // console.log(data);
+            assert.equal('brian', data.username );
+            assert.equal('Brian', data.firstName);
+            assert.equal('Sutton', data.lastName);
+        })
+        .catch(function(err){
+            console.log(err);
+        })
+        
+    });
```

Comment:
I don't see any mocking in the unit test, we should not talk to real database from our unit tests

Reply:
if there is an error in getting data from database this test is not failing its just logging error, can you please make sure the test fails